pages/date-range-picker/with-value.page.tsx

```diff
@@ -0,0 +1,80 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React, { useContext, useState } from 'react';
+
+import { Box, Checkbox, DateRangePicker, DateRangePickerProps, FormField, Link } from '~components';
+
+import AppContext from '../app/app-context';
+import ScreenshotArea from '../utils/screenshot-area';
+import {
+  DateRangePickerDemoContext,
+  dateRangePickerDemoDefaults,
+  generateI18nStrings,
+  generateRelativeOptions,
+  isValid,
+} from './common';
+
+export default function DatePickerScenario() {
+  const { urlParams, setUrlParams } = useContext(AppContext as DateRangePickerDemoContext);
+  const [value, setValue] = useState<DateRangePickerProps['value']>({
+    type: 'absolute',
+    startDate: '2018-01-09T12:34:56Z',
+    endDate: '2018-01-19T15:30:00Z',
+  });
+
+  const monthOnly = false;
+  const dateOnly = urlParams.dateOnly ?? dateRangePickerDemoDefaults.dateOnly;
+  const absoluteFormat =
+    urlParams.absoluteFormat ?? ('dateRangePickerDemoDefaults.absoluteFormat' as DateRangePickerProps.AbsoluteFormat);
+
+  return (
+    <Box padding="s">
+      <h1>Date range picker with selected date</h1>
+      <label>
+        Absolute format{' '}
+        <select
+          value={absoluteFormat}
+          onChange={event =>
+            setUrlParams({
+              absoluteFormat: event.currentTarget.value as DateRangePickerProps.AbsoluteFormat,
+            })
+          }
+        >
+          <option value="iso">Iso (Default)</option>
+          <option value="long-localized">Long localized</option>
+        </select>
+      </label>
+      <Checkbox
+        disabled={monthOnly}
+        checked={dateOnly}
+        onChange={({ detail }) => setUrlParams({ dateOnly: detail.checked })}
+      >
+        Date-only
+      </Checkbox>
+      <br />
+      <Link id="focus-dismiss-helper">Focusable element before the date range picker</Link>
+      <br />
+      <br />
+      <ScreenshotArea>
```

Comment:
This is the ScreenshotArea that will be used to continue testing for both default-date and default-date-only tests when the url with the right params are passed into this page